src/MBrace.Flow/CloudFlow.fs

```diff
@@ -1123,3 +1139,55 @@ module CloudFlow =
                         return! projection collector.Result
                 } }
 
+
+
+
+                
+    /// <summary>Creates a CloudFlow from the ReceivePort of a CloudChannel</summary>
+    /// <param name="channel">the ReceivePort of a CloudChannel.</param>
+    /// <returns>The result CloudFlow.</returns>
+    let ofCloudChannel (channel : IReceivePort<'T>) : CloudFlow<'T> =
```

Comment:
Yes, for both CloudFlow.ofArray and CloudFlow.ofCloudSource it feels very implicit that there is partitioning and/or multi-readers going on.  I was going to suggest an alternative name for CLoudFlow.ofArray (like CloudFlow.paritionArray) but wasn't sure what's the best choice.  Perhaps it's ok as is.

CloudFlow.ofCloudSource is even more implicit  as people may very likely think that a cloud source is sequential, being read one-by-one. I for one didn't realize that you just planned to have many workers reading off the same channel in order to partition - that's very cool and interesting.